bin/convert-argv.js

```diff
@@ -330,7 +334,7 @@ module.exports = function(yargs, argv, convertOptions) {
 		}, function() {
 			ensureArray(options, "plugins");
```

Comment:
Could you move the `ensureArray(options, "plugin")` into the addPlugin method to cleanup the code a bit.